src/test/java/com/conveyal/datatools/manager/controllers/api/FeedSourceControllerTest.java

```diff
@@ -255,20 +276,10 @@ public void createFeedSourceWithLabels() throws IOException {
     }
 
     /**
-     * Retrieve a feed source containing the deployed feed version.
+     * Retrieve the latest feed version for a feed source.
      */
     @Test
     void canRetrieveFeedSourceWithDeployedFeedVersion() throws IOException {
-        DeployJob deployJob = new DeployJob(
-            "Deploying " + deploymentDeployed.name,
-            deploymentDeployed,
-            user,
-            new OtpServer(),
-            "test-deploy",
-            DeployJob.DeployType.USE_PRELOADED_BUNDLE,
-            true
-        );
-        deployJob.run();
```

Comment:
It's not needed because I also removed the code related to this task from the DeployJob. Running this job no longer has and effect on defining the deployed feed version.